DESCRIPTION

```diff
@@ -1,29 +1,29 @@
 Package: hector
 Title: The Hector Simple Climate Model
 Version: 3.0.0
-Authors@R: c(person("Robert", "Link", 
```

Comment:
@bpbond needs to add Skylar for lo warming ratio work & carbon tracking work, who else needs to be added here? BVW

Reply:
Pushed 7a4d30645ebf9d1250875ad492772a4227648fa9 adding @skygering